packages/angular/src/generators/scam/schema.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "SCAMGenerator",
+  "cli": "nx",
+  "title": "SCAM Generator Options Schema",
+  "type": "object",
+  "description": "Creates a new, generic component definition in the given or default project.",
+  "additionalProperties": false,
+  "properties": {
```

Comment:
Some options that are supported by the `@schematics/angular:component` are missing here. Apart from `export`, `skipImport` and `module` which is fine not to be exposed since we handle them internally, the following are missing:

- flat
- prefix
- selector
- skipSelector
- type

I think we should support everything the original schematic supports except the ones that are needed to handle internally to prevent the regular NgModule behavior.

Is there a reason to exclude the above options?

Reply:
I thought they would impact the SCAM architecture, but thinking more about it, it should be fine